kits/expense-budget-copilot/apps/expense-budget-copilot/app/globals.css

```
@@ -0,0 +1,40 @@
+@tailwind base;
```

Stylelint (17.14.0)

[error] Unknown rule scss/at-rule-no-unknown. Did you mean at-rule-no-unknown?

(scss/at-rule-no-unknown)